app/models/kpi.rb

```diff
@@ -82,67 +82,61 @@ def self.api_imports
   def self.projects_with_ci
     projects_with_ci = Kpi::CiImporter.find_projects_with_ci
 
-    {
-      ci_present: convert_to_percentage(projects_with_ci),
-      ci_absent: convert_to_percentage(Project.count - projects_with_ci)
-    }
+    convert_to_percentage({
+      ci_present: projects_with_ci,
+      ci_absent: Project.count - projects_with_ci
+    })
   end
 
   def self.currently_active_products
     active_projects = Project.where(state: ACTIVE_STATUSES).where.not(last_commit_date: nil).count
```

Comment:
I think this would be better as an 'or':

Where status is active OR there is a commit in the last year.

We have some active projects for which there is no github repo :(